src/meta-srv/src/cluster.rs

```diff
@@ -50,16 +50,26 @@ pub struct MetaPeerClient {
 }
 
 impl MetaPeerClient {
-    // Get all datanode stat kvs from leader meta.
-    pub async fn get_all_dn_stat_kvs(&self) -> Result<HashMap<StatKey, StatValue>> {
+    async fn get_dn_key_value(&self, keys_only: bool) -> Result<Vec<KeyValue>> {
```

Comment:
Does changes to this file still necessary? Could we revert them?

Reply:
yes. It's necessary. we will get datanode number from this way.,